build/platforms.bzl

```diff
@@ -144,7 +144,7 @@ def _update_dict_for_platform_category(d, value, platforms, only_os = None):
 #
 # This macro produces a dictionary where each of the platform categories
 # (client, node, server, test, all) is enumerated and filled in
-# the the provided arguments as the values.
+# the provided arguments as the values.
```

Comment:
```suggestion
# with the provided arguments as the values.
```

Reply:
Thank for you suggestion.I already changed it.